src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -50,16 +50,16 @@ public void headerFallbackPolicy() {
 
   private String test1 = String.join("\n",
       "priority: t, s, c, b, a, m, g",
-      "fallback-policy: l no-loan r no-hold n basic-notice",
-      "m book cd dvd: l policy-a r request-1 n notice-1",
-      "m newspaper + g all: l policy-c r request-2 n notice-2",
-      "m streaming-subscription: l policy-c r request-3 n notice-3",
-      "    g visitor: l in-house r request-4 n notice-4",
-      "    g undergrad: l in-house r request-5 n notice-5",
-      "m book cd dvd + t special-items: l in-house r request-6 n notice-6",
-      "t special-items: l policy-d r request-7 n notice-7",
-      "    g visitor alumni: l in-house r request-8 n notice-8",
-      "a " + FIRST_INSTITUTION_ID + ": l in-house r request-8 n notice-8"
+      "fallback-policy: l no-loan r no-hold n basic-notice o overdue",
+      "m book cd dvd: l policy-a r request-1 n notice-1 o overdue-1",
+      "m newspaper + g all: l policy-c r request-2 n notice-2 o overdue-2",
+      "m streaming-subscription: l policy-c r request-3 n notice-3 o overdue-3",
+      "    g visitor: l in-house r request-4 n notice-4 o overdue-4",
+      "    g undergrad: l in-house r request-5 n notice-5 o overdue-5",
+      "m book cd dvd + t special-items: l in-house r request-6 n notice-6 o overdue-6",
+      "t special-items: l policy-d r request-7 n notice-7 o overdue-7",
+      "    g visitor alumni: l in-house r request-8 n notice-8 o overdue-8",
+      "a " + FIRST_INSTITUTION_ID + ": l in-house r request-8 n notice-8 o overdue-8"
```

Comment:
The FIRST_INSTITUTION_ID should not use 8 but 9: request-9 n notice-9 o overdue-9
Please update expected results accordingly.

Reply:
Yes, thanks. I did it. Moreover, I added this test case (regarding institution type) for each policy type.